src/class.rs

```diff
@@ -284,6 +284,28 @@ pub trait Class: Module {
     fn as_r_class(self) -> RClass {
         RClass::from_value(*self).unwrap()
     }
+
+    /// Remove the allocator function of a class.
+    ///
+    /// Useful for RTypedData, where instances should not be allocated by
+    /// the default allocate function.
```

Comment:
Could you note that most people won't need to call this explicitly, as the `#[derive(TypedData)]`/`#[wrap]` macros will call this for you?

Reply:
Done, feel free to adjust the wording.